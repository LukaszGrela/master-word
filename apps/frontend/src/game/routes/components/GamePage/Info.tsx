import { FC } from 'react';

export const Info: FC<{ text: string }> = ({ text }) => {
  return <p className="read-the-docs">{text}</p>;
};
