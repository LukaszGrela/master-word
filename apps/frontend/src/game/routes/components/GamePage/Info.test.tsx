import { describe, expect, it } from 'vitest';
import { render } from '@testing-library/react';
import { Info } from './Info';

describe('game', () => {
  describe('routes', () => {
    describe('components', () => {
      describe('GamePage', () => {
        describe('Info', () => {
          it('renders', () => {
            const { container } = render(<Info text="Info paragraph" />);
            expect(container).toMatchSnapshot()
          });
        });
      });
    });
  });
});
