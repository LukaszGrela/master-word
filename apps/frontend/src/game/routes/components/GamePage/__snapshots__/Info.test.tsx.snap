// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`game > routes > components > GamePage > Info > renders 1`] = `
<div>
  <p
    class="read-the-docs"
  >
    Info paragraph
  </p>
</div>
`;
